release-1.6/release-1.6.md

```diff
@@ -0,0 +1,42 @@
+(To Be Approved at upcoming community meeting)
+
+#Proposed timeline for 1.6
+Since Kubernetes 1.5 is set to release on Dec 12, 2016, and 
+December tends to have a lot of vacation time, we will begin the 1.6 release the first week of 2017.
+We will use the time between releases for planning, designing and bugfixes.
+
+
+The proposal below is identical in layout to the 1.4 and 1.5 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- some sigs will focus on stabilization for 1.6, and there may be fewer new
+  feature proposals
+- there are fewer major holidays and vacation breaks this quarter 
+
+##1.6 Overview
+- Dec 8 - Jan 3: planning/design/bugfix period
+- Jan 3 (Tues) coding start (7w)
+- Jan 24 (Tues) features repo freeze
+- Feb 27 (Mon) feature complete, move to bugfix (4w)
+- March 22 (Weds) release
+
+##1.6 Details
+
+###Jan 3 - Feb 27
+- 7 week coding period
+- Release 1.6 alphas every 2 weeks
+
+###Feb 27 - Mar 22
+- Enter code slush on head, no more features or major refactors
+- Fix bugs and run tests
+- Start Milestone Burndown meetings
+- Branch and cut Beta release on Mar 8 (Weds)
+
+###Mar 8 - Mar 22
+- Open head for 1.7 work on Mar 18, after 1.6 release branch has been fast-forwarded.
+- Fix bugs and run tests, update docs
+- Release 1.6 on March 22
+
+
+#Key features
+[Feature tracking spreadsheet link] 
```

Comment:
@grodrigues3 feel free to use this spreadsheet link - https://docs.google.com/spreadsheets/d/1nspIeRVNjAQHRslHQD1-6gPv99OcYZLMezrBe3Pfhhg/edit?usp=sharing.

It is empty at the moment, I'm in a process of clarifying the status of the features for 1.6 and will update it.